plugins/modules/identity/keycloak/keycloak_client.py

```diff
@@ -522,6 +522,16 @@
     client_id: test
     state: present
 
+- name: Create or update Keycloak client (minimal example), authentification with token
+  local_action:
+    module: keycloak_client
```

Comment:
The modern syntax is:
```suggestion
  delegate_to: localhost
  community.general.keycloak_client:
```